ktor-network/posix/interop/network.def

```diff
@@ -17,3 +22,17 @@ static inline void select_fd_clear(fd_set *set) {
 static inline int select_fd_isset(int descriptor, fd_set* set) {
     return FD_ISSET(descriptor, set);
 }
+
+#if defined(M_DARWIN)
+#include <arpa/inet.h>
+static inline const char *ktor_inet_ntop(int af, const void *src, char *dst, socklen_t size) {
+    return inet_ntop(af, src, dst, size);
+}
+#elif defined(MINGW)
+// TODO
```

Comment:
I am not familiar with the Windows C APIs, could you help me here, please? Is Kotlin/Native Windows even supported for the Ktor network module? Previously it was supported I think but it seems to have been removed in commit https://github.com/ktorio/ktor/commit/8b6f0742b03c9964d93365b86864f7bd4e2f9769. I do not need Windows support, so it is not an issue for me, but I would like to know what to do in this PR.

Reply:
Fixed: https://github.com/ktorio/ktor/pull/2129/commits/04557d2994ee659a5c46874c827c80cdd927a33c